gnat2goto/driver/tree_walk.adb

```diff
@@ -654,14 +634,18 @@ package body Tree_Walk is
       --    return temp_array;
       --  }
       function Build_Array_Lit_Func_Body (N : Node_Id) return Irep is
+
          Pos_Iter : Node_Id := First (Expressions (N));
          Source_Loc : constant Source_Ptr := Sloc (N);
          Bounds : constant Node_Id := Aggregate_Bounds (N);
          Low_Expr : constant Irep := Do_Expression (Low_Bound (Bounds));
          High_Expr : constant Irep := Do_Expression (High_Bound (Bounds));
          Index_Type_Node : constant Entity_Id := Etype (Etype (Bounds));
+         Index_Type : constant Irep := Do_Type_Reference (Index_Type_Node);
          Len_Expr : constant Irep :=
-           Make_Array_Length_Expr (Low_Expr, High_Expr, Index_Type_Node);
+           Build_Array_Size (First      => Low_Expr,
```

Comment:
Is `Build_Array_Size` not more or less the same as the old `Make_Array_Length_Expr`? Do we need 2 names for this?

Reply:
Yes, it does the same: I moved it to `goto_utils` and renamed to make it consistent with naming therein. We only have one now.